c/caarlos0-env/caarlos0-env_ubi8_3.sh

```diff
@@ -0,0 +1,32 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : caarlos0/env [caarlos0-env] 
+# Version       : v6.7.2
+# Tested on     : UBI 8.3
+# Script License: Apache-2.0 License    
+# Maintainer    : Varsha Aaynure <Varsha.Aaynure@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+yum update -y 
+
+#Install required files
+yum install -y golang
+
+#Cloning Repo
+go get github.com/caarlos0/env/v6
```

Comment:
Correct me if I am wrong.
Looks like, This always gets the latest V6 version (in future it would be v6.7.3 or v6.7.4).
It won't be specific to version v6.7.2.

So my suggestion is to go with `git clone` and `git checkout` the required version.
Also take the version as a variable, something like : `VERSION=${1:-v6.7.2}`